pkg/service/manifest/model/model.go

```diff
@@ -179,3 +178,11 @@ type Request struct {
 	// This is an output only field.
 	CredentialManifestJWT keyaccess.JWT `json:"credentialManifestJwt"`
 }
+
+type PresentationDefinitionRef struct {
+	// id of the presentation definition created with PresentationDefinitionAPI. Must be empty if `value` is present.
+	ID *string `json:"presentationDefinitionId"`
+
+	// value of the presentation definition to use. Must be empty if `id` is present.
```

Comment:
Is this enforced? if I put a id and presentationDefinition it will throw an error right?